contacts/helper.js

```diff
@@ -0,0 +1,70 @@
+import {
+  ID_CONSTANTS,
+  EMPTY,
+  ZERO,
+  ONE,
+  MINUS_ONE,
+  MAX_NUMBER_OF_LIKES,
+  MIN_NUMBER_OF_LIKES
+} from './constants';
+import {
+  Person
+} from './entities';
+
+
+// id generator
+// returns a string
+export const generateUniqueID = () => {
+  const s4 = () => Math.floor((1 + Math.random()) * 0x10000).toString(16).substring(1);
+  return `${s4()}-${s4()}-${s4()}-${s4()}`;
+};
+
+
+// we know that the last ID_LENGTH characters in htmlId related to a
+// person object is the id the of the the person object
+export const getPersonIdByHTMLElementId = htmlElementId => htmlElementId
+  .slice(htmlElementId.length - ID_CONSTANTS.ID_LENGTH);
+
+
+export const sortArrayAlphabetically = (arrayOfContacts) => {
+  arrayOfContacts.sort((person1, person2) => {
```

Comment:
theory says this could go like you did with the likes sorting:
arrayOfContacts.sort((p1, p2) => p1.fullName - p2.fullName))
or like this
arrayOfContacts.sort((a, b) => p1.fullName.localeCompare(p2.fullName))

if this works, and you use it in a single place, don't wrap it into another function, just use it like this where you need it